client/api/batch-client-proxy/fileProxy.ts

```diff
@@ -97,7 +97,7 @@ export default class FileProxy {
                         isDirectory: headers["ocp-batch-file-isdirectory"],
                         url: headers["ocp-batch-file-url"],
                         properties: {
-                            contentLength: headers["content-length"],
+                            contentLength: parseInt(headers["content-length"], 10),
```

Comment:
It is actually not the default. Default is 10 unless it starts with `0x`. But there was a warning telling that I should provide an argument